financial-templates-lib/clients/ExpiringMultiPartyClient.js

```diff
@@ -58,6 +66,15 @@ class ExpiringMultiPartyClient {
   // Whether the given undisputed `liquidation` (`getUndisputedLiquidations` returns an array of `liquidation`s) is disputable.
   // `tokenRedemptionValue` should be the redemption value at `liquidation.time`.
   isDisputable = (liquidation, tokenRedemptionValue) => {
+    if (!liquidation || !liquidation.numTokens || !liquidation.amountCollateral || !tokenRedemptionValue) {
+      this.logger.debug({
```

Comment:
seeing this is an invalid param, should this perhaps be a `warn` level event>

Reply:
agreed